### PR TITLE
Add doxy comments for file and classes, update license to 2020

### PR DESCRIPTION
**Pull request by apachano**
Just some quick Doxy, no functional changes.

If approved feel free to merge.

### CODE REVIEW

**Comment by beeperdeeper089** [comment]
Refer to my comment the issue you made for this (#50) let's see whether we actually need to do this.

**Comment by apachano** [comment]
@beeperdeeper089 There's still some doxy in there but I updated to match what Toby did.

File: Engine/Include/Quartz2/Settings.hpp [hunk 1]
```diff
@@ -26,6 +26,14 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+/**
+ * @file Settings.hpp
+ * @brief Implements a settings management system
+ * 
+ * @copyright Copyright (c) 2019-20 Genten Studios
```
Comment (beeperdeeper089) [inline Engine/Include/Quartz2/Settings.hpp]: We don't need the copyright twice in one file, the file header is enough I would say.

File: Engine/Include/Quartz2/Settings.hpp [hunk 2]
```diff
@@ -51,6 +66,13 @@ namespace q2
 
 	public:
 		Setting();
+		/**
+		 * @brief Construct a new Setting object
+		 * 
+		 * @param name The human readable name for the setting
```
Comment (beeperdeeper089) [inline Engine/Include/Quartz2/Settings.hpp]: In terms of syntax do we want to standardise using param[in] and param[out] and [in/out] or do we just wanna "ignore" them to some point?

**Pull request merged**